src/wranglerArtifactManager.ts

```diff
@@ -45,6 +45,8 @@ export type OutputEntryVersionUpload = z.infer<typeof OutputEntryVersionUpload>;
 const OutputEntryVersionUpload = OutputEntryBase.merge(
 	z.object({
 		type: z.literal("version-upload"),
+		/** The id of this uploaded version. Format is UUID . */
+		version_id: z.string().optional(),
```

Comment:
This looks like a UUID when running `wrangler` locally with `WRANGLER_OUTPUT_FILE_DIRECTORY` and looking at the output file, but I opted to simply parse it as a string because I assume it's not that important.

Reply:
I would go ahead and parse it as a UUID since it should only ever be a UUID according to https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/output.ts#L141

if the format changes from UUID, that would definitely be a breaking wrangler change!

Reply:
Sorry, I haven't had the time to get back to this. Is this PR otherwise acceptable if I change this to be parsed as an UUID?